packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -342,8 +332,9 @@ export default {
               displayName: collaborator.label,
               shareType: collaborator.value.shareType,
               permissions: bitmask,
+              role: this.selectedRole,
               expirationDate: this.expirationDate,
-              storageId
+              storageId: this.highlightedFile.fileId || this.highlightedFile.id
```

Comment:
Is storageId === highlightedFile.fileId? 🤔

Reply:
jup it contains storageId:resourceId